src/librustc/lib.rs

```diff
@@ -68,6 +68,7 @@ extern crate serialize as rustc_serialize; // used by deriving
 #[cfg(test)]
 extern crate test;
 
+#[allow(private_in_public)]
```

Comment:
Good point, for some reason I thought that the warnings weren't ok.
`pub extern crate` works fine except that it are not visible to other crates right now. That doesn't appear to be a problem for stage0 though.